src/test/java/io/jenkins/plugins/junit/checks/JUnitChecksPublisherTest.java

```diff
@@ -114,4 +115,32 @@ public void extractChecksDetailsFailingMultipleTests() throws Exception {
 
         assertThat(output.getTitle().get(), is("failed: 3, passed: 5"));
     }
+
+    @Test
+    @SuppressWarnings("OptionalGetWithoutIsPresent")
+    public void setCustomCheckName() throws Exception {
+        WorkflowJob j = rule.jenkins.createProject(WorkflowJob.class, "singleStep");
+        j.setDefinition(new CpsFlowDefinition("stage('first') {\n" +
+              "  node {\n" +
+              "    def results = junit(testResults: '*.xml', checksName: 'Custom Checks Name')\n" + // node id 7
```

Comment:
that comment is not needed here, (or above if you want to remove it)

```suggestion
              "    def results = junit(testResults: '*.xml', checksName: 'Custom Checks Name')\n" +
```